system/Database/Forge.php

```diff
@@ -405,23 +405,11 @@ public function addForeignKey($fieldName = '', string $tableName = '', $tableFie
     {
         $fieldName  = (array) $fieldName;
         $tableField = (array) $tableField;
```

Comment:
I moved this logic to _processForeignKeys(). I can't remember why exactly but it had to do with the new method processIndexes() which the calls _processForeignKeys().